spec/requests/atmosphere/api/clew_spec.rb

```diff
@@ -172,20 +167,29 @@ def cs
         expect(clew_at_response['appliance_types'][1]['matched_flavor']).to clew_flavor_eq  flavor
 
         #TODO complete this test to check all the returned fields
-
       end
+    end
+
+    it 'does not return AT with visible_to owner when user is not owner' do
```

Comment:
Not owned ATs were loaded because admin role was always turned on (off course for admins) for all crew requests. This was main issue for #176. This spec verifies that when not asked admin roles is not loaded in cancan abilities and not owned ATs are not returned.